syntax = "proto3";
package com.moon.demo.cs;

enum MsgID {
    PLACE_HOLDER = 0;
    MSG_ID_DEMO_1_REQ = 1000;
    MSG_ID_DEMO_1_RSP = 1001;
    MSG_ID_DEMO_2_REQ = 1002;
    MSG_ID_DEMO_2_RSP = 1003;
    MSG_ID_DEMO_3_REQ = 1004;
    MSG_ID_DEMO_3_RSP = 1005;
    MSG_ID_DEMO_4_REQ = 1006;
    MSG_ID_DEMO_4_RSP = 1007;
    MSG_ID_DEMO_5_REQ = 1008;
    MSG_ID_DEMO_5_RSP = 1009;
    MSG_ID_DEMO_6_REQ = 1010;
    MSG_ID_DEMO_6_RSP = 1011;
}
